Source/WebDriver/WebDriverService.cpp

```diff
@@ -838,6 +946,10 @@ void WebDriverService::connectToBrowser(Vector<Capabilities>&& capabilitiesList,
     }
 
     auto sessionHost = makeUnique<SessionHost>(capabilitiesList.takeLast());
+#if ENABLE(WEBDRIVER_BIDI)
+    // So we can close any living WebSocketConnection to the WebDriver client when the browser is killed.
```

Comment:
Ditto about usefulness here, this is self-evident in my opinion since the variables are well named.

Reply:
Removed.